renderers/photoaudiohaptics.schema.json

```diff
@@ -0,0 +1,91 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/renderers/photoaudiohaptics.schema.json",
+    "title": "Photo Audio Haptic Renderer",
+    "type": "object",
+    "description": "Data for basic audio haptics selection renderer ca.mcgill.a11y.image.renderer.PhotoAudioHaptics.",
+    "definitions": {
+        "centroid": {
+            "type": "array",
+            "items": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 1
+            },
+            "minItems": 2,
+            "maxItems": 2
+        },
+        "coordinates": {
```

Comment:
Not required since this is very single use, but previously we've encoded each coordinate pair as a tuple and then had longer lists of coordinate pairs be arrays of those tuples. This helps avoid problems with, for example, an odd number of coordinates being submitted by accident and having that pass validation.

Reply:
given that this could potentially require significant reworking would it be possible to address the other issues before we dedicate time to this?

Reply:
Sure, although note that that's the output format used by semantic segmentation currently so you would have the additional task of processing that.